bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import json
+import logging
+from pathlib import Path
+from typing import List, Tuple
+
+import discord
+import requests
```

Comment:
Since the discord.py bot framework is asynchronous, the requests library cannot be used. You can check out other cogs in this repository for examples on how to use [aiohttp](https://aiohttp.readthedocs.io/en/stable/) to make asynchronous web requests.

Reply:
Alright, will check that out as soon as possible. Thanks for the quick reply!

Reply:
@sco1 So should I just close this PR and start afresh?